src/copyright/agent/copyright.cc

```diff
@@ -131,7 +131,7 @@ int main(int argc, char** argv)
     {
       int uploadId = atoi(fo_scheduler_current());
 
-      if (uploadId == 0) continue;
+      if (uploadId <= 0) continue;
```

Comment:
Can you please tell me why this change was made ? in which case uploadId  goes to a negative value ?

Reply:
The call here can pass negative values
https://github.com/fossology/fossology/blob/master/src/scheduler/agent/agent.c#L242